client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -155,6 +164,38 @@ export default function CompositeCheckout( {
 		]
 	);
 
+	const validateDomainContact =
+		validateDomainContactDetails || wpcomValidateDomainContactInformation;
+
+	const renderDomainContactFields = (
+		domainNames,
+		contactDetails,
+		updateContactDetails,
+		applyDomainContactValidationResults
+	) => {
+		return (
+			<WPCheckoutErrorBoundary>
+				<ContactDetailsFormFields
+					countriesList={ countriesList }
+					contactDetails={ contactDetails }
+					onContactDetailsChange={ updateContactDetails }
+					onValidate={ ( values, onComplete ) => {
+						// TODO: Should probably handle HTTP errors here
+						validateDomainContact( values, domainNames, ( httpErrors, data ) => {
+							if ( data.messages ) {
```

Comment:
This is fine but five lines for a debug message seems like a lot. Maybe something like this would be a little more compact:

```js
debug( 'Domain contact info validation ' + data.messages ? 'errors' : 'successful', data.messages );
```

Reply:
Prettier expands this to four lines :/

(Albeit with only one expression, which is still an improvement)